llama_index/indices/managed/vectara/retriever.py

```diff
@@ -0,0 +1,150 @@
+"""Vectara index.
+An index that that is built on top of Vectara
+"""
+
+import logging
+from typing import Any, List, Optional, Sequence, Dict
+import json
+
+from llama_index.schema import TextNode
+from llama_index.indices.base_retriever import BaseRetriever
+from llama_index.schema import NodeWithScore
+from llama_index.indices.managed.base import ManagedIndex
+from dataclasses import dataclass
+from llama_index.schema import BaseNode
+
+_logger = logging.getLogger(__name__)
+
+
+@dataclass
+class VectaraQuery:
+    """Vectara query."""
+
+    similarity_top_k: int = 1
+    query_str: Optional[str] = None
+
+    # For Hybrid search: 0 = neural search only, 1 = bm25 only. In between values are a linear interpolation
+    lambda_val: Optional[float] = None
```

Comment:
Can you expand a bit more on this setting? If I set it to 0.25 for example, what does that mean?

Reply:
Added a reference. Essentially if you set it to say 0.25 it means it's provides more weight to the keyword search as opposed to neural search. What more detail would be helpful?

Reply:
OK that makes sense, thanks!